utilities/pipelines/staticValidation/module.tests.ps1

```diff
@@ -802,6 +802,38 @@ Describe 'Test file tests' -Tag 'TestTemplate' {
             }
         }
     }
+    Context 'Role Assignmnet test' {
+
+        $roleAssignmentTestCases = @()
+
+        foreach ($moduleFolderPath in $moduleFolderPaths) {
+           $rbacFilePath = Join-Path $moduleFolderPath '.bicep/nested_roleAssignments.bicep'
+            if (Test-Path $rbacFilePath) {
+                $nestedFileContent = Get-Content $rbacFilePath 
```

Comment:
I'd suggest to call it not `$nestedFileContent` but for example `$rbacFileContent` which would also match the `$rbacFilePath`